src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -220,9 +230,29 @@ class RtpReceiverImpl @JvmOverloads constructor(
                                     node(retransmissionRequester)
                                     node(paddingOnlyDiscarder)
                                     node(VideoParser(streamInformationStore, logger))
-                                    node(Vp8Parser(logger))
-                                    node(videoBitrateCalculator)
-                                    node(packetHandlerWrapper)
+                                    demux("Payload Type") {
```

Comment:
I wonder if this logic should just be part of `VideoParser` ?

Reply:
Hm, possibly.  Would we be easily able to use the node infrastructure then, though?

Reply:
I just meant the `VideoParser` node above.